device-types/HPE/EdgeLine-EL1000-10GBE2XSFPP-V2.yaml

```diff
@@ -0,0 +1,23 @@
+---
+manufacturer: HPE
+model: Edgeline EL1000
+slug: edgeline-el1000-880273-b21
+part_number: 880273-B21
+u_height: 2
+is_full_depth: false
+airflow: front-to-rear
+subdevice_role: parent
+comments: '[HPE Edgeline EL1000 Converged Edge System | Hardware Specs | HPE]https://www.hpe.com/psnow/doc/c05211199.html)'
+power-ports:
```

Comment:
It looks like this PSU is hot swappable, thus the power on this device should be under `module-bays`, and the associated PSU should be added to the module-types folder if it does not already exist